tools/report-parser/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
it doesn't look like this needs to be a package at all. Is it installed anywhere? If not, TBH I'd rather do something like:

- put contents of `report-parser.js` into `tools/pipelines/scripts/test-report-parser.js` (you'll need to make the scripts folder, but this is more sensible organization anyway imo)
- remove LICENSE, README, and package.json
- add a short doc comment at the top of the report generator file describing its purpose

Reply:
> it doesn't look like this needs to be a package at all. Is it installed anywhere? If not, TBH I'd rather do something like:
> 
> 
> 
> - put contents of `report-parser.js` into `tools/pipelines/scripts/test-report-parser.js` (you'll need to make the scripts folder, but this is more sensible organization anyway imo)
> 
> - remove LICENSE, README, and package.json
> 
> - add a short doc comment at the top of the report generator file describing its purpose

There's a scripts folder at the root, and I think that's a better location than creating a new one.

Is there any value to making this TS rather than JS? Not a leading question.